src/components/BaseStyles/BaseStylesService.ts

```diff
@@ -15,16 +15,38 @@
 
 import { css } from '@emotion/core';
 
+import { StyleProps } from '../../styles/styled';
 import { textMega } from '../../styles/style-helpers';
 
-export const createBaseStyles = theme => css`
+const FONTS_BASE_URL = 'https://static.sumup.com/fonts/latin-greek-cyrillic';
```

Comment:
Silly question here, is static.sumup.com also a CDN?

Reply:
Yes, I confirmed that it's behind Cloudfront.